### PR TITLE
Swipe delete icon

### PR DESCRIPTION
**Pull request by BDSchoeler**
Easy ui fix. I added larger icons and changed the padding.

### CODE REVIEW

**Review by samantha-kerr — APPROVED** [review]
Looks good to me

**Review by benrs — APPROVED** [review]

File: test/androidTest/java/org/thoughtcrime/securesms/database/AttachmentDatabaseTest.java [hunk 1]
```diff
@@ -47,7 +47,6 @@ public void testThumbnailGenerationTaskNotRunWhenThumbnailExists() throws Except
     // Works as the Future#get() call in AttachmentDatabase#getThumbnailStream() makes updating synchronous
     verify(database, never()).updateAttachmentThumbnail(any(MasterSecret.class), any(AttachmentId.class), any(InputStream.class), anyFloat());
   }
-
```
Comment (benrs) [inline test/androidTest/java/org/thoughtcrime/securesms/database/AttachmentDatabaseTest.java:50]: Shouldn't need this change.

**Review by jacobrs — APPROVED** [review]
Looks good!

**Review by lori-dalkin — APPROVED** [review]
Nice

**Pull request merged**